public/app/features/transformers/calculateHeatmap/utils.ts

```diff
@@ -1,7 +1,11 @@
 const { abs, round, pow } = Math;
 
-export function roundDec(val: number, dec: number) {
-  return round(val * (dec = 10 ** dec)) / dec;
+// https://stackoverflow.com/a/48764436
+// rounds half away from zero
+export function roundDec(val: number, dec = 0) {
+  let p = 10 ** dec;
+  let n = val * p * (1 + Number.EPSILON);
+  return round(n) / p;
```

Comment:
@ryantxu we should pull this out into a generic util fn and re-use, where shall it go?

Reply:
somethign in: https://github.com/grafana/grafana/tree/main/packages/grafana-data/src/utils